src/setup-gazebo-windows.ts

```diff
@@ -0,0 +1,43 @@
+import * as utils from "./utils";
+import * as conda from "./package_manager/conda";
+
+// List of mapped Gazebo distro to gz-sim versions
+const validLibVersions: { distro: string; libVersion: number }[] = [
+	{
+		distro: "garden",
+		libVersion: 7,
+	},
+	{
+		distro: "harmonic",
+		libVersion: 8,
+	},
+];
+
+/**
+ * Get gz-sim library version
```

Comment:
```suggestion
 * Get gz-sim library version
 *
 * The function returns the gz-sim version corresponding to a gz-$collection since conda does not currently support
 * the gz-$collection metapackages. See https://github.com/conda-forge/gz-sim-feedstock/issues/61
```